src/main/java/services/FileLoader.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2022
+ * Reads the file of initial values
+ * use it for free
+ */
+
+package services;
+
+import instances.Farm;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.LineIterator;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileLoader {
+
+    public static final char DIVIDER_CHAR = ':';
+    public static final String FILE_PATH = "src/main/resources/farm_resources.txt";
+    private static final List<String> list = new ArrayList<>();
+    private static final Logger log = LogManager.getLogger(Farm.class.getName());
+
+    /**
+     *  Names of variables for reading from the file
+     */
+    enum InitialResources {
+        MONEY("money"), LAND("land"), MEAT_PRICE("meat_price"), MILK_PRICE("milk_price");
```

Comment:
we list values of enum in a column

```suggestion
        MONEY("money"), 
        LAND("land"), 
        MEAT_PRICE("meat_price"), 
        MILK_PRICE("milk_price");
```